hassrelease/changelog.py

```diff
@@ -14,7 +14,9 @@
 IGNORE_LINE_LABELS = set(['reverted'])
 LABEL_HEADERS = {
     'new-platform': 'New Platforms',
+    'new-feature': 'New Features (incomplete)',
     'breaking change': 'Breaking Changes',
+    'cherry-picked': 'Fixes from beta',
```

Comment:
This will only work if we filter out PRs from different milestones (which I think we do but need to verify).

Otherwise we'll get 0.66 cherry picks in the 0.67 notes.

Reply:
They get filtered out earlier:
```py
if (pr.milestone is not None and
        pr.milestone.title != release.version):
    continue
```